common/types/admin.go

```diff
@@ -392,6 +394,18 @@ type IsolationGroupPartition struct {
 // Indicating that task processing isn't to occur within this isolationGroup anymore, but all others are ok.
 type IsolationGroupConfiguration map[string]IsolationGroupPartition
 
+func (i IsolationGroupConfiguration) ToPartitionList() []IsolationGroupPartition {
```

Comment:
a simple test can be added for this func

Reply:
I can, however this is covered in the test `TestIsDrainedHandler` so I'm unclear why it's not appearing as covered in the coverage...